### PR TITLE
(webdriver): fix custom request error message override

### PR DESCRIPTION
**Pull request by erwinheitzman**
## Proposed changes

I noticed that the error message from Chromedriver has changed so I have updated our code accordingly.

## Types of changes

- [ ] Polish (an improvement for an existing feature)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (improvements to the project's docs)
- [ ] Specification changes (updates to WebDriver command specifications)
- [x] Internal updates (everything related to internal scripts, governance documentation and CI files)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

### CODE REVIEW

File: packages/webdriver/src/utils.ts [hunk 1]
```diff
@@ -248,12 +248,21 @@ export class CustomRequestError extends Error {
         let errorMessage = errorObj.message || errorObj.class || 'unknown error'
 
         /**
-         * improve error message for Chrome and Safari on invalid selectors
+         * Improve Chromedriver's error message for an invalid selector
+         *
+         * Chrome:
+         *  error: 'invalid argument'
+         *  invalid 'invalid argument: invalid locator\n  (Session info: chrome=122.0.6261.94)'
```
Comment (christian-bromann) [inline packages/webdriver/src/utils.ts:255]:
```suggestion
         * message: 'invalid argument: invalid locator\n  (Session info: chrome=122.0.6261.94)'
```

**Review by christian-bromann — COMMENTED** [review]
One comment. Can you double check if all unit test pass? I haven't been able to get the build passing again but hopefully have some time today for this.

**Comment by christian-bromann** [comment]
Should we backport this to `v8`?

**Comment by erwinheitzman** [comment]
I don't think it's worth back porting as it is just a small improvement

**Comment by erwinheitzman** [comment]
Regarding the unit tests, the only tests I could see failing were the devtools tests

**Comment by christian-bromann** [comment]
> Regarding the unit tests, the only tests I could see failing were the devtools tests

Ok, I will take a look at them. Will move forward merging this PR. Thanks a lot!

**Comment by christian-bromann** [comment]
@erwinheitzman mind updating the branch?

**Review by christian-bromann — APPROVED** [review]
LGTM 👍

**Comment by christian-bromann** [comment]
@erwinheitzman is this something we would like to backport to v8?

**Comment by erwinheitzman** [comment]
We can but it's a very minor difference so I don't think it's worth the effort tbh. If you want I can do it though

**Comment by christian-bromann** [comment]
@erwinheitzman I was able to cherry-pick the commit and pushed it to `v8`

**Pull request merged**